### ISSUE TITLE
How to add json data in Design Time?

### ISSUE THREAD

**Issue by zpeeppy** [OYIon/LiveXAML#6]
Please support live json data in design time same Gorilla Player.
Thanks

**Comment by ionoy** [comment]
It should be possible to do, although since Xamarin Forms Live is not a previewer but an actual live app, it's a little bit more tricky. I'm thinking of adding a context menu to XAML files, something like "Apply design time data". This would open a text editor with a JSON file where you could define your mock model.

![image](https://user-images.githubusercontent.com/592624/28458192-8787a36c-6e11-11e7-9105-d972a1166982.png)

**Comment by zpeeppy** [comment]
Yes, good idea. It also can map json property for each Xaml Content Page.

**Comment by ionoy** [comment]
I don't think LiveXAML needs design time data at this point. No one else requested this feature, and LiveXAML works with live data, which takes away the need for design time data.

**Comment by FrancoisM** [comment]
No one requested design time data? How do you do the bindings at design time then?

**Comment by ionoy** [comment]
Why do you need Design Time with LiveXAML? The whole point is to see your Live data in a running application. Maybe I'm missing something here.

**Comment by FrancoisM** [comment]
I don't want to see live data when designing. I want to change data on the fly to see how the view accommodate to it. I want to play with all my visibility bindings and so on. I want to test edge case. Moreover, the design is done by non developers directly in xaml. They don't do anything but xaml, no C#.

The simplest case: an activity indicator when loading data. Say I want to work on its appearance. But the data is there (of course Live Xaml runs "Live") so the indicator is not visible. I have to "temporarily" play with bindings in xaml to remove the loaded data and customize the indicator but then I should not forget to put back bindings as they should be. 

I come from Gorilla Player. They have a json data file for design data. This is awesome. Why am I moving to LiveXaml (which I started to pay) from a free tool that I love you might ask? Well, they have no support whatsoever, they are not open source and the tool doesn't work as soon as you have custom cell in a dedicated xaml file whereas LiveXaml does.

Actually now that I write these lines I realize that LiveXaml is not what I'm after. I don't want to only modify the xaml page I navigated to in my app. I want to DESIGN any page at any time. Basically I want a better Gorilla Player and I misled myself thinking LiveXaml was that when clearly they are two different products. You definitely have a great product, keep up with the good work, but this is not the one I'm after.
